atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/DeletePaxosLog.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2015 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.cli.command;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import com.palantir.atlasdb.config.LeaderConfig;
+
+import io.airlift.airline.Command;
+
+@Command(name = "deletePaxosLog", description = "Delete the paxos log after restoring at least quorum leaders that lost their acceptor log")
+public class DeletePaxosLog extends AbstractCommand {
+
+    @Override
+    public Integer call() {
+
+        // get leader config
+        LeaderConfig leaderConfig;
+        try {
+            leaderConfig = scm.provideAtlasDbConfig().leader().get();
+        } catch (IllegalStateException e) {
+            System.err.println("Error: Config file is missing required leader block configuration.");
+            return 1;
+        }
+
+        try {
+            for(File log : leaderConfig.acceptorLogDir().listFiles()) {
+                Files.delete(log.toPath());
+            }
+            for(File log : leaderConfig.learnerLogDir().listFiles()) {
+                Files.delete(log.toPath());
+            }
```

Comment:
I am okay with this as long as it's clear to the user that if you accidentally delete a given boxes Paxos logs they NEED to go and run the cli below.  I would consider this important enough to include in the cli's output / help.